package.json

```diff
@@ -40,6 +41,7 @@
     "typescript": "^3.7.2"
   },
   "dependencies": {
+    "restify-errors": "^8.0.2",

```

Comment:
This should go in packages/sdk/package.json, not here. Same for @types above.

Reply:
Good call, done.